src/Microsoft.Azure.ServiceBus/ServiceBusConnectionStringBuilder.cs

```diff
@@ -14,12 +15,20 @@ public class ServiceBusConnectionStringBuilder
         const char KeyValueSeparator = '=';
         const char KeyValuePairDelimiter = ';';
         static readonly string EndpointScheme = "amqps";
-        static readonly string EndpointFormat = EndpointScheme + "://{0}.servicebus.windows.net";
         static readonly string EndpointConfigName = "Endpoint";
         static readonly string SharedAccessKeyNameConfigName = "SharedAccessKeyName";
         static readonly string SharedAccessKeyConfigName = "SharedAccessKey";
         static readonly string EntityPathConfigName = "EntityPath";
 
+        string entityPath, sasKeyName, sasKey, endpoint;
+
+        /// <summary>
+        /// Instantiates a new <see cref="ServiceBusConnectionStringBuilder"/>
+        /// </summary>
+        public ServiceBusConnectionStringBuilder()
```

Comment:
@nemakam @vinaysurya from a public API point of view, since this is a public class, it's better to seal/internalize members/methods, unless known for sure those need to be public. So 👎 for keeping it as-is (i.e. `public`) if there's no real reason for that.